pulse_actions/handlers/treeherder_add_new_jobs.py

```diff
@@ -62,12 +66,26 @@ def on_event(data, message, dry_run, treeherder_host, acknowledge, **kwargs):
                                                                           treeherder_link))
 
     LOG.info("New jobs requested by %s for %s" % (requester, treeherder_link))
-    LOG.info("List of builders:")
-    for b in buildernames:
+    LOG.info("List of requested jobs:")
+    for b in requested_jobs:
         LOG.info("- %s" % b)
 
+    # Handle TC tasks separately
+    task_labels = [x for x in requested_jobs if x.startswith('TaskLabel==')]
+    buildernames = list(set(requested_jobs) - set(task_labels))
+
     buildernames = filter_invalid_builders(buildernames)
 
+    # Scheduling TaskCluster jobs
+    # Make sure that decision task id is not null and task_labels are there to schedule
+    if task_labels and len(decision_task_id):
+        try:
+            schedule_action_task(decision_task_id=decision_task_id,
```

Comment:
This handler can receive the value dry_run==True.

You can see we use it on line 104:
https://github.com/mozilla/pulse_actions/pull/82/files#diff-3347eec26da4d6c21c012e608b6eb6c2R104

This allow us to run pulse_actions in --dry-run mode for local development.

I would like you to pass that parameter to this function so we can have that functionality.